scripts/ci/selective_ci_checks.sh

```diff
@@ -117,6 +127,7 @@ function output_all_basic_variables() {
     if [[ ${FULL_TESTS_NEEDED_LABEL} == "true" ]]; then
         initialization::ga_output postgres-exclude '[{ "python-version": "3.6" }]'
         initialization::ga_output mysql-exclude '[{ "python-version": "3.7" }]'
+        initialization::ga_output mssql-exclude '[{ "python-version": "3.7" }]'
```

Comment:
Ok. I found the reason. You need to add empty array as `mssql-exlude` in else few lines below.

Reply:
@aneesh-joseph ^^ one more thing :)

Reply:
thank you @potiuk 👍